pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md

```diff
@@ -70,298 +71,510 @@ us-west1
+      dcos_variant = "open"
 
-  # dcos_variant              = "ee"
-  # dcos_license_key_contents = "${file("./license.txt")}"
-  dcos_variant = "open"
+      dcos_install_mode = "${var.dcos_install_mode}"
+    }
 
-  dcos_install_mode = "${var.dcos_install_mode}"
-}
+    output "masters-ips" {
+      value       = "${module.dcos.masters-ips}"
+    }
 
-output "masters-ips" {
-  value       = "${module.dcos.masters-ips}"
-}
+    output "cluster-address" {
+      value       = "${module.dcos.masters-loadbalancer}"
+    }
 
-output "cluster-address" {
-  value       = "${module.dcos.masters-loadbalancer}"
-}
+    output "public-agents-loadbalancer" {
+      value = "${module.dcos.public-agents-loadbalancer}"
+    }
+    ```
 
-output "public-agents-loadbalancer" {
-  value = "${module.dcos.public-agents-loadbalancer}"
-}
-```
+2) Re-run the new execution plan with the changes you have made to the `main.tf`configuration file.  
 
-2) The changes are made to `main.tf`, now re-run the new execution plan.  
+    ```bash
+    terraform plan -out=plan.out
+    ```
 
-```bash
-terraform plan -out=plan.out
-```
+    This step ensures that the state is stable and confirms that you can create the resources necessary to scale the private agents to the desired number. Executing the plan adds the following resources as a result of scaling up the cluster’s private agents:
+    - One instance resource
+    - Two null resources which handle the DC/OS installation and prerequisites in the background.
 
-This step ensures that the state is stable and to confirm that you will create the resources necessary to scale the private agents to the desired number.
+    You should see a message similar to the following:
+<p align=center><img src="./images/scale/terraform-plan.png" /></p>
 
-<p align=center>
-<img src="./images/scale/terraform-plan.png" />
-</p>
+3) Run the following command to have Terraform deploy the new set of resources:
 
-After executing the plan, you should see a message similar to above message. There will be `three` resources added as a result of scaling up the cluster’s private agents (1 instance resource & 2 null resources which handle the DC/OS installation and prerequisites in the background).
+    ```bash
+    terraform apply plan.out
+    ```
 
-3) Now, that the plan is set, get Terraform to build/deploy the new set of resources.
+    You should see an output similar to the following:
+  <p align=center><img src="./images/scale/terraform-apply.png" /></p>
+  
+4) Check your DC/OS cluster using the DC/OS UI to verify the additional agents have been added. 
 
-```bash
-terraform apply plan.out
-```
+    You should see `four` total nodes connected. For example:
+  <p align=center><img src="./images/scale/node-count-4.png" /></p>
 
-<p align=center>
-<img src="./images/scale/terraform-apply.png" />
-</p>
+# Upgrading the cluster
+Terraform also makes it easy to upgrade the DC/OS cluster to a newer version of DC/OS.
```

Comment:
Mesosphere Universal Installer makes upgrading your DC/OS cluster to newer patches & major versions of DC/OS easy and effortless.  In this tutorial, we will upgrade our cluster from DC/OS 1.11.7 to 1.12.0.